### PR TITLE
visidata 0.98.1 (new formula)

### PR DESCRIPTION
**Pull request by anjakefala**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by anjakefala** [comment]
A new version had been released since I had opened the PR, so I attempted to update the formula, but I am not really certain why the build has failed. I did not see these errors on my sierra OS and I do not completely understand them.

**Comment by anjakefala** [comment]
Oh, it seems like you are deprecating the 'depends_on :python3'?

I have not heard anything, so I ended up [building a tap here](https://github.com/saulpw/homebrew-vd) to host the formula. 

If you can provide any feedback on the formula, it would be appreciated. I can help you update the python-formula documentation.

**Comment by anjakefala** [comment]
ping @ilovezfs, is it possible to get an update on whether to close the PR, whether you need more information or if you would like me to wait longer?

File: Formula/visidata.rb [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    venv.pip_install resources
+    venv.pip_install_and_link buildpath
+    man1.install "visidata/man/vd.1"
+    Language::Python.each_python(build) do |_, version|
+      bundle_path = libexec/"lib/python#{version}/site-packages"
+      bundle_path.mkpath
+      ENV.prepend_path "PYTHONPATH", bundle_path
+      (lib/"python#{version}/site-packages/homebrew-visidata-bundle.pth").write "#{bundle_path}\n"
```
Comment (ilovezfs) [inline Formula/visidata.rb]: I don't think we should be doing this. As a rule, Homebrew packages Python applications not Python libraries, so exposing the libraries here doesn't make sense.

File: Formula/visidata.rb [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    end
+  end
+
+  test do
+    system "#{libexec}/bin/python3", "-c", "import visidata"
```
Comment (ilovezfs) [inline Formula/visidata.rb]: The test should exercise and verify the functionality of `vd`.

File: Formula/visidata.rb [hunk 3]
```diff
@@ -0,0 +1,82 @@
+class Visidata < Formula
+  include Language::Python::Virtualenv
+  desc "Terminal utility for exploring and arranging tabular data"
+  homepage "http://visidata.org"
+  url "https://files.pythonhosted.org/packages/49/95/15af273f829a97944baf15f6fc36b427a0bff5917913d87d3cd01a4582ee/visidata-1.0.tar.gz"
+  sha256 "79ad431f6bc1c4df7f8ecd8b26421d03f49f186c4e883b73310ffc5747b5a653"
+  head "https://github.com/saulpw/visidata.git"
```
Comment (ilovezfs) [inline Formula/visidata.rb]: would rather we not include HEAD

File: Formula/visidata.rb [hunk 4]
```diff
@@ -0,0 +1,82 @@
+class Visidata < Formula
+  include Language::Python::Virtualenv
+  desc "Terminal utility for exploring and arranging tabular data"
+  homepage "http://visidata.org"
```
Comment (ilovezfs) [inline Formula/visidata.rb]: trailing slash

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by anjakefala** [comment]
I updated the formula based on your requests. The good news is that the test seems to be functional. ^^; I will investigate the failures.

**Comment by anjakefala** [comment]
The failures are due to the fact that VisiData, since VisiData was running headless it tried to check whether something was piped in through stdin (which it was not).

Code updates will be required for VisiData in order for it to run in headless mode in this scenario.

**Comment by anjakefala** [comment]
We have decided that, since there is not a reliable way to fix this issue for the homebrew tests, to continue to just maintain our own tap.

Closing PR.

**Pull request closed without merging**